flatpak-proxy.c

```diff
@@ -2294,7 +2275,7 @@ queue_fake_message (FlatpakProxyClient *client, GDBusMessage *message, ExpectedR
   Buffer *buffer;
 
   client->last_serial++;
-  client->serial_offset++;
+  g_assert (client->last_serial > G_MAXUINT32 / 2);
```

Comment:
There's at least 3 places using `G_MAXUINT32 / 2 `in this patch, maybe worth its own define ?

Reply:
Fixed.